YvesIraguha/personalFinanceApi
# Set up Development Environment

#### What does this PR do?
- setup the project development environment
#### Description of Task to be completed?
- setup linting
- set up transpiling
- integrate with Pivotal tracker and Hound
#### How should this be manually tested?
- Clone the repo and checkout the branch `ch-set-environment-167564572`
- Run `npm install` to install all dependencies
- You should be good to go. 
#### What are the relevant pivotal tracker stories?
### 167564572
